### PR TITLE
Begin tuning auto

### PR DESCRIPTION
**Pull request by calebeby**

### CODE REVIEW

**Comment by calebeby** [comment]
@BrendanBurkhart can you add the `max_speed` changes to `rotate()` and `forward()` here as well?

**Comment by BrendanBurkhart** [comment]
In this pr?

File: autonomous/rotate.py [hunk 1]
```diff
@@ -11,6 +11,6 @@ class Rotate(AutonomousStateMachine):
 
     @state(first=True)
     def rotate(self):
-        if self.drivetrain.rotate(degrees=-90):
+        if self.drivetrain.rotate(degrees=-90, max_speed=90):
```
Comment (calebeby) [inline autonomous/rotate.py]: Why 90?

Comment (brendanburkhart) [reply autonomous/rotate.py]: You had it set to something like 1.5 radians/second which is roughly 90 degrees/second, more like 85 I believe. I can change it to 85 so the tuning is right where you left it if you want.

Comment (calebeby) [reply autonomous/rotate.py]: Oh, ok.

**Comment by calebeby** [comment]
OK, I'm not going to merge this until I finish tuning it tomorrow.

**Review by brendanburkhart — APPROVED** [review]

**Pull request merged**